src/Faker/Calculator/Ean.php

```diff
@@ -17,11 +17,9 @@ class Ean
      *
      * @see https://en.wikipedia.org/wiki/International_Article_Number
      *
-     * @param string $digits
-     *
      * @return int
      */
-    public static function checksum($digits)
+    public static function checksum(string $digits)
```

Comment:
I wonder who uses these classes directly.

Perhaps they should be marked as `@internal`?